### PR TITLE
Release 2.5.0 PowerScale

### PR DESCRIPTION
**Pull request by felixs88**
# Description
* Manage server certificate on power scale array including import, update, default, and remove certificates.
* The info module is enhanced to list all the server certificates
* Example playbook addition for certificate module and info module
* UT script for certificate module
* RST file generated for certificate and info module
* Updated documentation changes related to Release 2.5.0

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [x] I have performed Ansible Sanity test using --docker default
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

### CODE REVIEW

File: docs/Release Notes.md [hunk 1]
```diff
@@ -65,25 +65,20 @@ The Ansible Modules for Dell PowerScale support the following features:
...
-- The settings module has been enhanced to support this functionality:
-    - Added support for getting and modifying cluster owner information and cluster identity information.
+- The server certificate module supports this functionality:
+    - Added support for importing, modifying, setting defaults, and deleting server certificates.
 - The Info module has been enhanced to support this functionality.
-    - Added support for listing SMB global settings, detailed network interfaces, NTP servers, email settings, cluster identity, cluster owner and SNMP settings through info module.
+    - Added support for listing server certificates.
 
 Known issues
 ------------
```
Comment (Bhavneet-Sharma) [inline docs/Release Notes.md:84]: Update version below in this file: https://github.com/dell/ansible-powerscale/pull/72/files#diff-dbb84812da420b1d22115c2206447d6f70be5e618d263ed43d3a2518f0bbe692R127

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
[the remark this answers is not in the document]
Comment (felixs88) [reply .gitignore]: Yes, removed the empty line

File: docs/Release Notes.md (hunk 1 above)
Comment (felixs88) [reply docs/Release Notes.md:84]: Yes, updated the 2.4.1 --> 2.5.0 version

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by Jennifer-John — DISMISSED** [review]

**Review by meenakshidembi691 — APPROVED** [review]

File: docs/ISSUE_TRIAGE.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Should explain what happened, what was expected and how to reproduce it together
 
  - Ansible Version: [e.g. 2.15]
  - Python Version [e.g. 3.11]
- - Ansible modules for Dell PowerScale Version: [e.g. 2.4.1]
+ - Ansible modules for Dell PowerScale Version: [e.g. 2.5.0]
  - PowerScale SDK version: [e.g. isilon-sdk]
```
Comment (trisha-dell) [inline docs/ISSUE_TRIAGE.md:47]: PowerScale SDK version: [e.g. 0.3.0.1]

File: README.md [hunk 4]
```diff
@@ -67,51 +68,52 @@ This table provides information about the software prerequisites for the Ansible
 
 | **Ansible Modules** | **OneFS Version** | **Python version** | **Python SDK version** | **Ansible**              |
 |---------------------|-----------------------|--------------------|----------------------------|--------------------------|
-| v2.4.1 | 9.3.x <br> 9.4.x <br> 9.5.x | 3.9 <br> 3.10 <br> 3.11 | 9.3.0 <br> 9.4.0 <br> 9.5.0 | 2.14 <br> 2.15 <br> 2.16 |
+| v2.5.0 | 9.3.x <br> 9.4.x <br> 9.5.x | 3.9 <br> 3.10 <br> 3.11 | 9.3.0 <br> 9.4.0 <br> 9.5.0 | 2.14 <br> 2.15 <br> 2.16 |
```
Comment (trisha-dell) [inline README.md:71]: change SDK version to 0.3.0.1 instead of 9.3.0 <br> 9.4.0 <br> 9.5.0

Comment (felixs88) [reply README.md]: updated

File: docs/ISSUE_TRIAGE.md (hunk 3 above)
Comment (felixs88) [reply docs/ISSUE_TRIAGE.md]: updated

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Review by Jennifer-John — APPROVED** [review]

**Review by meenakshidembi691 — APPROVED** [review]

**Pull request merged**